packages/gallery/src/components/item/itemView.js

```diff
@@ -783,13 +782,14 @@ class ItemView extends React.Component {
 
 
   getItemAriaLabel() {
```

Comment:
why is this a function in the view element(react), can this move to galleryItem / itemsHelper?
in galleryItem we could have this as part of the calculatedAlt